defs/run_in_workspace.bzl

```diff
@@ -23,12 +23,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-BASE=$(pwd)
-cd $(dirname $(readlink {root_file}))
```

Comment:
can we expand this at build time?

Reply:
This code path is no longer relevant.

Reply:
if we could expand it at build time we could produce a file that worked in more situations?

Reply:
An issue is that it changes depending on whether the script is inside bazel run or called directly.

I suspect there's something exotic we can do, but I think we can do it in a follow up PR and this is an improvement over the current state of affairs.